src/Invictus.Testing.Tests.Integration/LogicAppAuthenticationTests.cs

```diff
@@ -49,18 +49,9 @@ public async Task AuthenticateLogicAppManagement_UsingAccessToken_Succeeds()
         {
             // Arrange
             string subscriptionId = Configuration.GetAzureSubscriptionId();
-            string tenantId = Configuration.GetAzureTenantId();
-            string clientId = Configuration.GetAzureClientId();
-            string clientSecret = Configuration.GetAzureClientSecret();
-
-            string authority = $"https://login.windows.net/{tenantId}";
-
-            var authContext = new AuthenticationContext(authority);
-            var credential = new ClientCredential(clientId, clientSecret);
-
-            AuthenticationResult token = await authContext.AcquireTokenAsync("https://management.azure.com/", credential);
+            string accessToken = Environment.GetEnvironmentVariable("AzureManagementApiAccessToken");
```

Comment:
Can you follow our existing approach with configuration, please:
https://github.com/invictus-integration/testing-framework/blob/master/src/Invictus.Testing.Tests.Integration/appsettings.json#L1

Feel free to add it there and tokenized with name in your script.

Reply:
done! however the pipeline is not valid due to the change in AzureSubscription:

> The pipeline is not valid. Job IntegrationTests: Step AzureCLI input connectedServiceNameARM references service connection Azure - Codit-Arcus which could not be found. The service connection does not exist or has not been authorized for use. For authorization details, refer to https://aka.ms/yamlauthz.

does a service connection exist or do you need to authorize the pipeline manually? 
Pipeline link in Azure DevOps: [link](https://dev.azure.com/codit/Invictus%20OSS/_build/results?buildId=48456&view=results)